docs/workitems.md

```diff
@@ -0,0 +1,114 @@
+# TFX Work Item commands
+
+Create, query and view [Work Items](https://www.visualstudio.com/en-us/docs/work/backlogs/add-work-items).
+
+## Get started
```

Comment:
Not sure where, but I'd call out (maybe a markdown > block quote) the problem you mentioned yesterday --- error that is shown when users try to use a non-collection level URL.